gun-violence-dashboard/src/pages/Homepage.js

```diff
@@ -1,22 +1,55 @@
 import React, { useEffect, useState } from "react";
 import USAMap from "react-usa-map";
 import axios from 'axios';
+import {
```

Comment:
a general note, I think the loading icon should remain until all of the data is loaded; there is still some jumping around as data loads after the load icon disappears

Reply:
Do you have some suggestions on how to resolve this? I am awaiting all the data, but it still jumps.